### PR TITLE
Use Kubernetes latest release 1.20

### PR DESCRIPTION
**Pull request by fmuyassarov**
Use Kubernetes latest release 1.20 and switch to CRI-O as for the container-runtime on the target cluster.

### CODE REVIEW

File: vm-setup/roles/v1aX_integration_test/vars/main.yml [hunk 1]
```diff
@@ -17,9 +17,10 @@ NODE_DRAIN_TIMEOUT: "{{ lookup('env', 'NODE_DRAIN_TIMEOUT') | default('0s', true
 BMOPATH: "{{ lookup('env', 'BMOPATH') }}"
 IRONIC_DATA_DIR: "{{ lookup('env', 'IRONIC_DATA_DIR') }}"
 KUBECONFIG_PATH: "/home/ubuntu/.kube/config"
-KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.19.3', true) }}"
-
-KUBERNETES_BINARIES_VERSION: "{{ lookup('env', 'KUBERNETES_BINARIES_VERSION') | default(lookup('env', 'KUBERNETES_VERSION') | default('v1.18.0', true), true) }}"
+# (FIXME) Add a lookup info front of KUBERNETES_VERSION and 
+# KUBERNETES_BINARIES_VERSION vars to inherit values from lib/common.sh
+KUBERNETES_VERSION: "v1.19.3"
+KUBERNETES_BINARIES_VERSION: "v1.19.3"
```
Comment (jan-est) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:23]: Why version 1.19.3 is still used here?

Comment (fmuyassarov) [reply vm-setup/roles/v1aX_integration_test/vars/main.yml:23]: That's for the target cluster. Right now I am trying with the source cluster first.

**Comment by fmuyassarov** [comment]
It seems there is some instability in the infra.
/test-v1a3-centos-integration

File: vm-setup/roles/v1aX_integration_test/tasks/move.yml [hunk 2]
```diff
@@ -17,6 +17,8 @@
        - ironic-endpoint-keepalived
        - ironic-log-watch
        - ironic-inspector-log-watch
+    become: yes
+    become_user: root
```
Comment (kashifest) [inline vm-setup/roles/v1aX_integration_test/tasks/move.yml:21]: Why do we need this ?

**Review by kashifest — APPROVED** [review]
lgtm. One small question inline

File: vm-setup/roles/v1aX_integration_test/tasks/move.yml (hunk 2 above)
Comment (fmuyassarov) [reply vm-setup/roles/v1aX_integration_test/tasks/move.yml:21]: When I did my local tests Ansible failed right here due to permissions, requiring me root permission. I know that it was working fine up until now without root, but I think it is not going to hurt. I can remove it and rerun the tests.

Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/tasks/move.yml:21]: ok, lets keep it then.

**Pull request merged**